### PR TITLE
Ability to add undocumented options to requests

### PR DESCRIPTION
**Pull request by tmconnors**
According to the wiki only `Conference, Dial, and Pause` have undocumented options. However, for consistency should we add this option to all TwiML verbs - in my opinion no.

### CODE REVIEW

File: src/main/java/com/twilio/twiml/Conference.java [hunk 1]
```diff
@@ -128,6 +134,9 @@ public String apply(ConferenceEvent event) {
     @XmlValue
     private final String name;
 
+    @XmlAnyAttribute
+    private Map<QName, String> options;
```
Comment (efossier) [inline src/main/java/com/twilio/twiml/Conference.java]: Will the XML serializer work serializing the key to a QName?

File: src/main/java/com/twilio/twiml/Conference.java [hunk 2]
```diff
@@ -224,6 +234,16 @@ public String getName() {
         return name;
     }
 
+    public Map<String, String> getOptions() {
+        Map<String, String> convertedMap = new HashMap();
```
Comment (jingming) [inline src/main/java/com/twilio/twiml/Conference.java]: should be using the `<>` here?

File: src/main/java/com/twilio/twiml/Conference.java (hunk 1 above)
Comment (tmconnors) [inline src/main/java/com/twilio/twiml/Conference.java]: @efossier Yes. I specifically choose `QName` for this reason.

**Comment by efossier** [comment]
+1

**Pull request merged**